### PR TITLE
Add test for bounded floor

### PR DESCRIPTION
**Pull request by cabanier**
There was no good test that showed how bounded floor was used.
This sample will show the local bounds and if the UA supports unbounded, it will switch to different spaces with their own bounds.

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
Why do you have both “local-floor” and “bounded-floor” as required features? Isn’t it redundant to have “local-floor” since you will be using “bounded-floor”?

**Comment by cabanier** [comment]
> Why do you have both “local-floor” and “bounded-floor” as required features? Isn’t it redundant to have “local-floor” since you will be using “bounded-floor”?

No, they are subtly different. 
The origin of `local-floor` is the (0,-floorheight,0) in the coordinate space that starts when initializing webxr or holding down the oculus/meta button.
The origin of `bounded-floor` is the center of the guardian boundary.

`local-floor` is used as the coordinate space of the experience while `bounded-floor` is used to place the flower and the bounds.

**Comment by cabanier** [comment]
@AdaRoseCannon ?

**Comment by AdaRoseCannon** [comment]
That makes sense, I haven't had time to test yet I can do so next week. But I happy for it to be merge anyway.

**Comment by cabanier** [comment]
> That makes sense, I haven't had time to test yet I can do so next week. But I happy for it to be merge anyway.

It's broken in current browser but should work in upcoming 34.3

**Review by toji — APPROVED** [review]
LGTM, and I think you can merge it as-is. The only thing I would comment on is that the sample explicitly creates an AR session, while I suspect that a bounded floor space will most commonly be used with a VR session? It would be nice if the sample could handle either as a way of emphasizing that the feature isn't necessarily tied to either session mode.

But again, I don't think that's a blocking issue.

File: tests/index.html [hunk 1]
```diff
@@ -154,6 +154,10 @@ <h2 class='tagline'>Test Pages</h2>
             { title: 'Exit Button', category: 'Tasty Burritos',
               path: 'exit-button.html',
               description: 'A scene with an in-world exit button.' },
+
+            { title: 'Bounded Floor', category: 'Fried Chicken',
```
Comment (toji) [inline tests/index.html:158]: 🍗

**Pull request merged**